--- Portfile.orig	2014-05-08 09:51:08.000000000 -0400
+++ Portfile	2014-05-08 13:39:48.000000000 -0400
@@ -6,7 +6,7 @@
 
 name                LyX
 conflicts           LyX1
-version             2.0.6
+version             2.0.8
 set branch          [join [lrange [split ${version} .] 0 1] .]
 categories          aqua
 license             GPL-2+
@@ -32,11 +32,10 @@
                     ftp://ftp.ntua.gr/pub/X11/LyX/stable/${branch}.x/ \
                     ftp://ftp.lyx.org/pub/lyx/stable/${branch}.x/ \
                     http://gd.tuwien.ac.at/publishing/tex/lyx/stable/${branch}.x/
+checksums           rmd160  6ab1e15ade2100a75020ef85b42f7a346f5ebe63 \
+                    sha256  b4bd32cf5f2c374b1ff88b69b53db0a2ba840942df0c3886b295fb2020683b5e
 
-checksums           rmd160  8e0e3af80726f566fdd8498f7410c36cffa6bdda \
-                    sha256  a5cb2bedee6e154b50881acd2316cf9d79162d6754e3aa159a6d579f5a828ad2
-
-patchfiles          patch-configure.diff \
+patchfiles          patch-types.diff \
                     patch-libc++_compliance.diff
 
 configure.pre_args  --prefix=${applications_dir}/LyX.app
@@ -63,6 +62,10 @@
     default_variants +python27
 }
 
+variant debug description {Build with debugging information} {
+    configure.args-append   --enable-debug
+}
+
 post-configure {
     reinplace -E \
         "s|mkdir -p \\\$\\(pkgdatadir\\)|\
--- /dev/null	2014-05-08 13:48:42.000000000 -0400
+++ files/patch-types.diff	2014-05-08 13:32:34.000000000 -0400
@@ -0,0 +1,11 @@
+--- src/support/docstring.h.orig	2014-05-08 13:30:22.000000000 -0400
++++ src/support/docstring.h	2014-05-08 13:31:40.000000000 -0400
+@@ -24,7 +24,7 @@
+  * Use std::string only in cases 7-bit ASCII is to be manipulated
+  * within the variable.
+  */
+-typedef std::basic_string<char_type> docstring;
++//typedef std::basic_string<char_type> docstring;
+ 
+ /// Creates a docstring from a C string of ASCII characters
+ docstring const from_ascii(char const *);
--- files/patch-configure.diff	2014-05-08 09:50:41.000000000 -0400
+++ /dev/null	2014-05-08 13:52:21.000000000 -0400
@@ -1,10 +0,0 @@
---- configure.orig	2010-11-14 05:58:06.000000000 -0500
-+++ configure	2011-01-30 20:56:40.000000000 -0500
-@@ -8746,6 +8746,7 @@
- 		QT4_LIB=`$PKG_CONFIG --libs-only-l QtCore QtGui`
- 
- 		LIBS="$LIBS `$PKG_CONFIG --libs-only-other QtCore QtGui`"
-+		LIBS="$LIBS -framework Carbon -framework AppKit"
- 	fi
- 	PKG_CONFIG_PATH=$save_PKG_CONFIG_PATH
- 
